site/content/entry/yuzu-progress-report-may-2020/index.md

```diff
@@ -0,0 +1,196 @@
+much time in the macro interpreter. This should be a global performance boost independent of GPU vendor or API.
+[Pull Request #4009.](https://github.com/yuzu-emu/yuzu/pull/4009)
+
+When [Rodrigo](https://github.com/ReinUsesLisp) improved yuzu’s ASTC decoding in the past, he also added a rule to use native 
+hardware decoding whenever possible. The Nvidia driver informs such capabilities, but they use an internal software decoder, 
+and turns out, it is far slower than our own implementation. Ignoring the Nvidia driver level software decoder gave a massive 
+performance improvement when facing the dreaded ASTC texture format in games. It will still be immediate on Intel GPUs,
+no software optimizations will beat a dedicated hardware decoder.
+[Pull Request #4014.](https://github.com/yuzu-emu/yuzu/pull/4014)
+
+## Bug fixes and improvements
+
+Vulkan is an ongoing process in yuzu, and it has stability problems as expected of a relatively new and complex feature. 
+[Blinkhawk](https://github.com/FernandoS27) did a couple of critical changes to Vulkan and Asynchronous GPU, improving stability 
+considerably.
+[Pull Request #3905.](https://github.com/yuzu-emu/yuzu/pull/3905)
+
+Speaking about Vulkan, many 2D games have their sprites flipped or completely wrong, but once again, we currently lack the 
+extension required to fix this, so [Rodrigo](https://github.com/ReinUsesLisp) implemented support for `NV_viewport_swizzle`, 
+this Nvidia only extension is the only way to solve this problem in a clean manner for now. An universal method is being 
+developed.
+[Pull Request #3885.](https://github.com/yuzu-emu/yuzu/pull/3885)
+
+{{< single-title-imgs
+    "Quack (Duck Game)"
+    "./10.png"
+    "./11.png"
+  >}}
+
+The updated libraries also gave us a new version of our [cubeb](https://github.com/kinetiknz/cubeb) audio engine which adds 
+support for 6 channel audio, [ogniK](https://github.com/ogniK5377) added the support for surround sound with this change.
+[Pull Request #3827.](https://github.com/yuzu-emu/yuzu/pull/3827)
+
+Our good shark [ogniK](https://github.com/ogniK5377) also fixed keyboard emulation support, so expect compatible games to be 
+able to have proper direct input from your keyboards now.
+[Pull Request #3926.](https://github.com/yuzu-emu/yuzu/pull/3926)
+
+[Morph](https://github.com/Morph1984) implemented the missung support for `R8G8UI` textures, fixing both the performance problems and saving crashes `The Walking Dead` games experienced.
+[Pull Request #3839.](https://github.com/yuzu-emu/yuzu/pull/3839)
+
+{{< single-title-imgs
+    "Thank you Toxa for the screenshot (The Walking Dead: The Final Season)"
+    "./08.png"
+  >}}
+
+One small but important annoyance when running a fullscreen program is the mouse being in front of everything you are doing. 
+[Tobi](https://github.com/FearlessTobi) implemented an option to automatically hide the mouse once it has been inactive after 
+some time.
```

Comment:
Change to `Although objectively a small issue, the mouse cursor didn't hide when running yuzu in full screen, causing a subjectively significant annoyance. Thankfully, [Tobi](https://github.com/FearlessTobi) implemented an [option to automatically hide the mouse](https://github.com/yuzu-emu/yuzu/pull/3892) once it has been inactive after some time.`